v2/emailpassword/common-customizations/signin-form/additional-field-props.mdx

```diff
@@ -0,0 +1,164 @@
+---
+id: additional-field-props
+title: Additional Field Props
+hide_title: true
+---
+
+<!-- COPY DOCS -->
+<!-- ./emailpassword/common-customizations/signin-form/additional-field-props.mdx -->
+
+import FrontendSDKTabs from "/src/components/tabs/FrontendSDKTabs"
+import TabItem from '@theme/TabItem';
+
+# Additional Field Props
+
+* [Modify Labels and Placeholder Text](#modify-labels-and-placeholder-text)
+* [Setting Default Values](#setting-default-values)
+* [Changing Optional Error Message](#changing-optional-error-message)
+
```

Comment:
add one more item to change field order: https://github.com/supertokens/supertokens-auth-react/issues/296#issuecomment-1236119677